app/src/other/document_categories/view/documentCategoryCrudDialog.html

```diff
@@ -0,0 +1,33 @@
+<md-dialog aria-label="{{ 'DOCUMENT_CATEGORIES.CREATE_EDIT_DOC_CATEGORY' | translate }}">
+  <form name="docCategoryForm">
+    <md-toolbar>
+      <div class="md-toolbar-tools">
+        <h1>{{ 'DOCUMENT_CATEGORIES.CREATE_EDIT_DOC_CATEGORY' | translate }}</h1>
+        <span flex></span>
+        <md-button class="md-icon-button" ng-click="dc.cancel()">
+          <md-icon aria-label="Close dialog"></md-icon>
+        </md-button>
+      </div>
+    </md-toolbar>
+
+    <md-dialog-content style="max-width:40em;max-height:410px; ">
+      <md-input-container>
+        <label>{{ 'DOCUMENT_CATEGORIES.NAME' | translate }}</label>
+        <input type="text" ng-model="dc.documentCategory.name" required ng-pattern="/^[a-zA-Z0-9_\-,!@#$%^&()=+ ]+$/" ng-disabled="dc.documentCategory.systemType">
```

Comment:
I guess this regEx pattern could be extracted somewhere into angular constants to avoid duplication. The documentTypeCrudDialog uses the same pattern.